src/index.ts

```diff
@@ -62,6 +64,19 @@ const PKG = require('../../package.json');
 const v2 = require('./v2');
 const {grpc} = new gax.GrpcClient();
 
+export interface GetInstancesCallback {
+  (
+    err: ServiceError | null,
+    result?: Instance[] | null,
+    nextQuery?: {} | null,
+    response?: google.bigtable.admin.v2.IListInstancesResponse | null
+  ): void;
+}
+export type GetInstancesResponse = [
+  Instance[] | null,
```

Comment:
I don't think this will ever be null, if the request fails the promise would get rejected.